src/components/Menu.js

```diff
@@ -39,55 +68,79 @@ const MenuSidebar = styled.div.attrs({
   className: "bg-yellow",
 })`
   height: 100vh;
-  width: ${props => (props.active ? "30vw" : "0")};
   transition: 0.5s;
-  // transition-delay: 0.1s;
+  width: ${props => (props.active ? "30vw" : "0")};
+  ${breakpoint.s`
+    height: 149px;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    width: ${props => (props.active ? "100vw" : "0")};
+  `};
 `
 const MenuMain = styled.div.attrs({
   className: "pv6 mv5",
 })`
-  height: 100vh;
-  width: ${props => (props.active ? "70vw" : "0")};
+  height: calc(100vh - 149px);
   transition: 0.5s;
-  // transition-delay: 0.1s;
+  width: ${props => (props.active ? "70vw" : "0")};
+  ${breakpoint.s`
+    position: fixed;
+    z-index: 15;
+    top: 149px;
+    left: 0;
+    width: 100vw;
+    padding-top: 0;
+    margin-top: 45px;
+  `};
 `
 
 const MenuNumber = styled.div.attrs({})`
   padding-top: 3px;
   font-size: ${({ active }) => (active ? "16px" : "0px")};
   transition: 0.5s;
-  // transition-delay: 0.1s;
+  ${breakpoint.s`
+    font-size: ${({ active }) => (active ? "12px" : "0px")};
+  `}
 `
 
 const Logo = styled.img.attrs({
   className: "border-box pa5-ns",
 })`
   width: ${({ active }) => (active ? "100%" : "0")};
   transition: 0.5s;
-  // transition-delay: 0.1s;
+  ${breakpoint.s`
+    height: 80px;
+  `}
 `
 
 const MenuImage = styled.div.attrs({
   className: "pl3 ttu",
 })`
-  font-size: ${({ active }) => (active ? "48px" : "0px")};
   transition: 0.5s;
-  // transition-delay: 0.1s;
-
-  -webkit-text-stroke: 1px black;
-  color: white;
-
-  &:hover {
+  ${breakpoint.ns`
+    font-size: ${({ active }) => (active ? "48px" : "0px")};
+    color: white;
+    -webkit-text-stroke: 1px black;
+    &:hover {
+      color: black;
```

Comment:
```suggestion
      color: var(--black);
```
will enable we stick to our variable colours